lib/apis/fetch.js

```diff
@@ -11,7 +11,15 @@ export default (url, options = {}) => {
 
     request(url, opts, (err, response) => {
       if (!!err || response.statusCode !== 200) {
-        return reject(err || new Error(response.body));
+        if (err) return reject(err);
```

Comment:
Some errors use the callback; some don't. In this case we'll get valid responses which are 4xx errors; so we just check the status code instead. I agree: it's weird.